openedx/core/djangoapps/plugins/plugin_contexts.py

```diff
@@ -0,0 +1,35 @@
+from importlib import import_module
+
+from logging import getLogger
+from . import constants, registry
+
+log = getLogger(__name__)
+
+
+def get_plugins_view_context(project_type, view_name, existing_context={}):
+    """
+    Returns a dict of additonal view context. Will check if any plugin apps
+    have that view in their context_config, and if so will call their
+    selected function to get their context dicts.
+    """
+    aggregate_context = {}
+
+    for app_config in registry.get_app_configs(project_type):
+        context_function_path = _get_context_function(app_config, project_type, view_name)
+        if context_function_path:
+            module_path, _, name = context_function_path.rpartition('.')
+            context_function = getattr(import_module(module_path), name)
+            plugin_context = context_function(existing_context)
```

Comment:
You'll want to defend against `import_module ` and `getattr` returning empty values in the previous line.

Reply:
Fixed!